e2e/tests/components/column-picker/column-picker.e2e-spec.ts

```diff
@@ -67,44 +120,32 @@ describe('Column Picker Tests', () => {
     });
 
     it('should allow moving a deselected column', async () => {
-        expect(await page.getColumnCount(ColumnPickerList.Deselected)).toBe(18);
-        expect(await page.getColumnCount(ColumnPickerList.Selected)).toBe(6);
         await page.selectColumn(ColumnPickerList.Deselected, 0);
         await page.selectBtn.click();
         expect(await page.getColumnCount(ColumnPickerList.Deselected)).toBe(17);
         expect(await page.getColumnCount(ColumnPickerList.Selected)).toBe(7);
-        expect(await page.getDeselection()).toBe('[ "Category", "Date Created", "Date Modified", "Department", "Document ID", "Flag", "From", "Icon", "Importance", "Location", "Location ID", "Message", "Organization", "Time", "Time Created", "Time Modified", "Work Completed" ]');
```

Comment:
This should probably not be removed.

Reply:
The issue with this is that the deselected always stores all items and the component just hides and reveals the relevant one. So this list never changes. The assertion of the counts combined with the selected column will be a clear enough indication it is working correctly. Let me know if you disagree - alternatively I can write something that checks a list of column names exist in the deselected list.

Reply:
No I think that's fine, but if that JSON is not being used in the tests then it should be removed from the test page.

Reply:
It is still used for the initial check so I think it can stay.